package.json

```diff
@@ -1,27 +1,29 @@
 {
     "author": "Unscrambl",
     "bin": {
-        "browserDriverInstaller": "./index.js",
-        "checkEnvironmentAndInstall": "./check-environment-and-install"
+        "installBrowserDrivers": "./install_browser_drivers"
     },
     "dependencies": {
-        "chai": "4.1.2",
-        "sinon-chai": "3.2.0",
-        "commander": "2.15.1",
-        "detect-mocha": "0.1.0",
-        "mocha": "5.1.1",
-        "readable-stream": "3.4.0",
-        "shelljs": "0.8.1",
-        "sinon": "5.0.2"
+        "commander": "^3.0.2",
+        "extract-zip": "^1.6.7",
+        "request": "^2.88.0",
+        "shelljs": "^0.8.3",
+        "tar": "^5.0.5"
+    },
+    "devDependencies": {
+        "chai": "^4.2.0",
+        "mocha": "^6.2.1",
+        "sinon": "^7.5.0",
+        "sinon-chai": "^3.3.0"
     },
     "description": "Installs Chrome and Gecko drivers that matches with the specified browser versions",
     "license": "Apache-2.0",
     "main": "index.js",
     "name": "browser-driver-installer",
     "repository": "github:unscrambl/browser-driver-installer",
     "scripts": {
-        "postinstall": "./postinstall",
-        "test": "node_modules/mocha/bin/mocha *.spec.js"
+        "postinstall": "./install_browser_drivers",
+        "test": "node_modules/mocha/bin/mocha *.spec.js --timeout 60000"
     },
-    "version": "1.0.32"
+    "version": "2.0.0"
```

Comment:
as soon as this package is published, all of the current installations will try to install this version and give errors. We should not tag this version as `latest` while publishing

Reply:
why, @mzyil?

Reply:
I was planning to update infra to use `npx installBrowserDrivers` instead but it won't work for the release branches that still use `npx checkEnvironmentAndInstall` from older infra versions.

I've added back `"checkEnvironmentAndInstall": "./install_browser_drivers"` as well for backward compatibility.